.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
+    target-branch: main
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 2
+    ignore:
+      - dependency-name: "*"
```

Comment:
Since this is in an `ignore` section, wouldn't that ignore _all_ depenencies and thereby effectively disable the dependabot PRs? But maybe I'm just reading this wrong.

Reply:
I got this from an issue
See https://github.com/dependabot/dependabot-core/issues/2219

As far as I understood:
- you need to specify a dependency-name e.g. `lodash` and then tell dependabot which semver type to ignore
- but we want all dependencies

Reply:
Wondering if we also want to ignore patch updates.